src/chatd.cpp

```diff
@@ -1837,6 +1845,7 @@ void Chat::disable(bool state)
 Idx Chat::getHistoryFromDb(unsigned count)
 {
     assert(mHasMoreHistoryInDb); //we are within the db range
+    mOldestIdxInDb = mDbInterface->getOldestIdx();
```

Comment:
Why do we need to update the oldest index in DB every time we load more history from DB? Does it change from a call to another? The only possible change I can imagine it that we loaded more messages backwards in history, and the index should get updated upon reception of the message, IMHO.